R/styles.R

```diff
@@ -1,5 +1,11 @@
-
-#' @importFrom crayon red green make_style bold
+rcmdcheck_color <- function(f) {
+  function(...) {
+    if (identical(Sys.getenv("RCMDCHECK_COLOR", "false"), "true")) {
+      withr::local_options(c(crayon.enabled = TRUE))
```

Comment:
As written this would only allow you to enable color, not disable it, not sure if we want that functionality or not...

Reply:
I updated this as well, now you can set the number of colors.